common/app/routes/challenges/components/video/Video.jsx

```diff
@@ -58,13 +61,14 @@ export class Video extends React.Component {
   render() {
     const {
       title,
+      blockName,
       shouldShowQuestions
     } = this.props;
-
+    let titleString = challengeTitle({ blockName, title });
```

Comment:
`blockName` would be the same inside each block, i.e `HTML5 and CSS` or `Basic JavaScript`. 

'title` is the challenge title, i.e `Say Hello to HTML Elements` or `Comparisons with the OR Operator`